app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
+
+    cp_command = files_names[0]
+    first_file = files_names[1]
+    second_file = files_names[2]
+
```

Comment:
Extract the command and file names using unpacking. But first check that length command.split is 3